packages/inter-protocol/test/psm/gov-add-psm-permit.json

```diff
@@ -0,0 +1,52 @@
+{
+  "consume": {
+    "agoricNamesAdmin": true,
+    "bankManager": true,
+    "board": true,
+    "chainStorage": true,
+    "zoe": "zoe",
+    "feeMintAccess": "zoe",
+    "economicCommitteeCreatorFacet": "economicCommittee",
+    "chainTimerService": "timer"
+  },
+  "produce": {
+    "$ISSUE": "These names assume there can be only one psm",
+    "psmCreatorFacet": true,
+    "psmAdminFacet": true,
+    "psmGovernorCreatorFacet": true
+  },
+  "installation": {
+    "consume": {
+      "contractGovernor": true,
+      "psm": true,
+      "mintHolder": true
+    }
+  },
+  "instance": {
+    "consume": {
+      "economicCommittee": true
+    },
+    "produce": {
+      "$ISSUE": "These names assume there can be only one psm",
```

Comment:
Are these intended to remain?

Reply:
no.

Reply:
hence https://github.com/Agoric/agoric-sdk/pull/6142#issuecomment-1237629267 and #6146 and such.